src/indieweb_utils/webmentions/discovery.py

```diff
@@ -71,29 +74,51 @@ def _discover_endpoints(url, headers_to_find):
     :return: The discovered endpoints.
     :rtype: dict[str, str]
     """
-    response = {}
+    response: Dict[str, str] = {}
 
     endpoint_request = requests.get(url)
 
-    http_link_headers = endpoint_request.headers.get("link")
+    response.update(_find_links_in_headers(headers=endpoint_request.headers, target_headers=headers_to_find))
```

Comment:
Refactored the implementation of this function to separate concerns.

Reply:
Looks good!